src/medUtilities/medUtilities.cpp

```diff
@@ -52,7 +52,7 @@ void medUtilities::copyMetaDataIfEmpty(medAbstractData* derived, medAbstractData
 
 void medUtilities::copyMetaDataIfEmpty(medAbstractData* derived, medAbstractData* original, QString metaDataKey)
 {
-    if (!derived->hasMetaData(metaDataKey))
+    if (!derived->hasMetaData(metaDataKey) && original->hasMetaData(metaDataKey))
```

Comment:
we had warning in pipelines for instance for each metadatakey which was not in "original"